.github/workflows/nix.yml

```diff
@@ -31,3 +31,17 @@ jobs:
       - name: Build and run a static polarity exe
         if: matrix.os == 'ubuntu-latest'
         run: nix -Lv run .#polarity-static -- --help
+
+  check_nix_devshell:
+    name: Check nix development shell
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: cachix/install-nix-action@v30
+      - name: Set up cachix
+        uses: cachix/cachix-action@v15
+        with:
+          name: polarity
+          authToken: ${{ secrets.CACHIX_AUTH_TOKEN }}
```

Comment:
```suggestion
```

These don't have to run in a new job.

Reply:
True! I just thought that these jobs test for rather different things.
But let's save some CPU cycles in caching.

Reply:
By accident :)

sorry

Reply:
np, I force-pushed your suggested change